pkg/kubernetes/testdata/install-input-no-manifests.yaml

```diff
@@ -0,0 +1,6 @@
+install:
+  - kubernetes:
+      description: "Install Hello World App"
+      manifests:
```

Comment:
Is this test file correct? It doesn't seem to match the name because there are manifests specified.